### PR TITLE
Add self-service registration info

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Add self-service registration to page and change headings to accommodate both Okta-hosted and self-hosted widgets
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-333065](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md [hunk 1]
```diff
@@ -18,4 +20,18 @@ config['authScheme'] = 'SESSION';
 
 4. Click **Save and Publish**.
 
+### Configure self-service registration in an embedded or self-hosted widget
+
+Before you can configure self-service registration in your embedded or self-hosted widget, make sure [self-service registration](/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/) is already enabled. Then you can set `feature.registration` in the widget to "true".
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md:25]: present tense update

Before you can configure self-service registration in your embedded or self-hosted widget, make sure **that you first enable** [self-service registration](/docs/guides/set-up-self-service-registration/configure-self-service-registration-policy/). **You can then set** `feature.registration` in the widget to `true`.

**Review by susanharper-okta — COMMENTED** [review]
couple comments

**Review by christinechoy-okta — COMMENTED** [review]
Andrea Skouras reviewed and approved the content.

File: packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md (hunk 1 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md:25]: Before you can configure self-service registration in your embedded or self-hosted widget, make sure that you first enable self-service registration. You can then set feature.registration in the widget to true.

File: packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md [hunk 2]
```diff
@@ -18,4 +20,18 @@ config['authScheme'] = 'SESSION';
...
+
+```json
+      },
+      features: {
+        // Used to enable registration feature on the widget.
+        // https://github.com/okta/okta-signin-widget#feature-flags
+         registration: true // REQUIRED
+      }
+```
+See the [Okta Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/) for more information on customizing the widget.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/customize-link-in-widget/index.md:35]: See the [Okta Sign-In Widget guide](/code/javascript/okta_sign-in_widget/).

**Review by susanharper-okta — COMMENTED** [review]
my comments

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**